x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx

```diff
@@ -48,6 +63,18 @@ export function getExpressionRenderer(coreSetup: CoreSetup<MapsPluginStartDepend
         // ignore error, lack of EMS file layers will be surfaced in dimension editor
       }
 
+      const renderComplete = () => {
+        const originatingApp = extractOriginatingApp(handlers.getExecutionContext());
```

Comment:
What causes originatingApp to be a value other then 'lens'? I created a new dashboard and created a by value lens visualization and the originatingApp is still lens. Is this expected?

Why does uiCounter id change if the originatingApp changes?

Reply:
@nreese thank you, you are right! I'll update it soon